lib/services/ios-debug-service.ts

```diff
@@ -72,14 +74,15 @@ class IOSDebugService implements IDebugService {
         }).future<void>()();
     }
 
-    private emulatorDebugBrk(): IFuture<void> {
+    private emulatorDebugBrk(shouldBreak? : boolean): IFuture<void> {
```

Comment:
I don't see many other methods with such signature. Can you point some? In fact most methods are using signature similar to what I've made here.